src/Data/Vec/Properties.agda

```diff
@@ -477,6 +488,11 @@ map-⊛ f g (x ∷ xs) = cong (f x (g x) ∷_) (map-⊛ f g xs)
 ++-injective ws xs eq =
   (++-injectiveˡ ws xs eq , ++-injectiveʳ ws xs eq)
 
+++-assoc : ∀ .(eq : (n + m) + o ≡ n + (m + o)) (xs : Vec A n) (ys : Vec A m) (zs : Vec A o) →
```

Comment:
I think it would be better for the order of length indices to be `m` `n` `o`, ie.
`++-assoc : ∀ .(eq : (m + n) + o ≡ m + (n + o)) (xs : Vec A m) (ys : Vec A n) (zs : Vec A o) →`
(But check the style guide as well)

Reply:
Done! Thanks for noticing this.